packages/block-library/src/more/edit.js

```diff
@@ -74,6 +75,7 @@ export default class MoreEdit extends Component {
 						size={ inputLength }
```

Comment:
I guess the size is unnecessary here right?

Reply:
If you adopt the way adding the style, I agree.

Reply:
Let's remove that and ship the PR.